vllm/distributed/parallel_state.py

```diff
@@ -28,7 +28,11 @@
 from unittest.mock import patch
 
 import torch
+import torch.distributed
 from torch.distributed import Backend, ProcessGroup
+from torch.distributed.distributed_c10d import (_get_pg_default_device,
```

Comment:
This import is not ideal. However we either had to do this or copy a bunch of helper functions from torch

Reply:
I'm fine with it.